recipe/meta.yaml

```diff
@@ -1,29 +1,27 @@
-{% set version = "0.8.0" %}
+{% set version = "0.10.0" %}
 
 package:
   name: stdlib-list
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/stdlib-list/stdlib-list-{{ version }}.tar.gz
-  sha256: a1e503719720d71e2ed70ed809b385c60cd3fb555ba7ec046b96360d30b16d9f
+  url: https://pypi.io/packages/source/s/stdlib_list/stdlib_list-{{ version }}.tar.gz
+  sha256: 6519c50d645513ed287657bfe856d527f277331540691ddeaf77b25459964a14
 
 build:
   number: 0
-  # trigger 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  skip: True  # [py<37]
 
 requirements:
-  build:
+  host:
     - python
+    - flit-core >=3.2,<4
     - pip
     - setuptools
     - wheel
```

Comment:
```suggestion
```
`setuptools` and `wheel` are not needed when `flit-core` is used to build a project.

Reply:
I assumed as much but noticed other recipes left both even though `flit-core` was being used. Thanks!